conf/app.routes

```diff
@@ -10,8 +10,11 @@ GET        /services                                    uk.gov.hmrc.cataloguefro
 GET        /libraries                                   uk.gov.hmrc.cataloguefrontend.repository.RepositoriesController.allLibraries
 GET        /prototypes                                  uk.gov.hmrc.cataloguefrontend.repository.RepositoriesController.allPrototypes
 
-GET        /create-a-repository                         uk.gov.hmrc.cataloguefrontend.createarepository.CreateARepositoryController.createARepositoryLanding()
-POST       /create-a-repository                         uk.gov.hmrc.cataloguefrontend.createarepository.CreateARepositoryController.createARepository()
+GET        /create-a-repository                         uk.gov.hmrc.cataloguefrontend.createarepository.CreateARepositoryController.createAServiceRepositoryLanding()
```

Comment:
can we change the url to `/create-service` (do we need the `-a-` in urls and classes etc.?)